cafebabel/templates/articles/tags/detail.html

```diff
@@ -31,7 +31,13 @@
 
 {% block body_class %}tag-page{% endblock %}
 {% block body %}
-  <div class=tag-header data-image={{ tag.image_url }}></div>
+  {% if tag.has_image %}
+    <div class="tag-header image">
+    <img src={{ tag.image_url or '' }} alt="Cover picture for #{{ tag.name }}">
+  {% else %}
+    <div class="tag-header default">
+  {% endif %}
+    </div>
```

Comment:
From my experience with templating, it's easier on the long run to have if clauses encapsulating the whole `div` for small parts:

```
    {% if tag.has_image %}
      <div class="tag-header image">
        <img src={{ tag.image_url or '' }} alt="Cover picture for #{{ tag.name }}">
      </div>
   {% else %}
      <div class="tag-header default"></div>
   {% endif %}

```

Reply:
ok